content/en/continuous_integration/setup_tests/junit_upload.md

```diff
@@ -10,87 +10,89 @@ further_reading:
-yarn global add @datadog/datadog-ci
+npm install -g @datadog/datadog-ci
 {{< /code-block >}}
 
 ## Uploading test reports
 
-To upload your JUnit XML test reports to Datadog:
+To upload your JUnit XML test reports to Datadog, run the following command, specifying the name of the service or library that was tested using the `--service` parameter, and one or more file paths to either the XML report files directly or directories containing them:
 
 {{< code-block lang="bash" >}}
-datadog-ci junit upload [--service] [--tags] [--max-concurrency] [--dry-run] <paths>
+datadog-ci junit upload --service <service_name> <path> [<path> ...]
 {{< /code-block >}}
 
-For example:
+You will need to specify a valid [Datadog API key][3] in the `DATADOG_API_KEY` environment variable, and the environment where tests were run (e.g. `local` when uploading results from a developer workstation, or `ci` when uploading them from a CI provider) in the `DD_ENV` environment variable. For example:
 
 {{< code-block lang="bash" >}}
-datadog-ci junit upload --service my-service \
-  --tags key1:value1 --tags key2:value2 \
-  unit-tests/junit-reports acceptance-tests/junit-reports e2e-tests/single-report.xml
+export DATADOG_API_KEY=a5f...ea170229
+export DD_ENV=ci
+datadog-ci junit upload --service my-api-service unit-tests/junit-reports e2e-tests/single-report.xml
 {{< /code-block >}}
 
+## Additional configuration settings
+
+This is the full list of options available when using the `datadog-ci junit upload` command:
+
 `--service`
-: The name of the service you're uploading JUnit XML reports for.<br>
-**Default**: the `DD_SERVICE` environment variable value.
+: Name of the service or library under test.<br/>
+**Environment variable**: `DD_SERVICE`<br/>
+**Default**: (none)<br/>
+**Example**: `my-api-service`
 
 `--tags`
-: An array of key-value pairs in the form `key:value`. This sets global tags applied to all spans. The resulting dictionary is merged with the content of the `DD_TAGS` environment variable. If a `key` appears both in `--tags` and `DD_TAGS`, the value in `DD_TAGS` takes precedence.
+: Key-value pairs in the form `key:value` to be attached to all tests (the `--tags` parameter can be specified multiple times). When specifying tags using `DD_TAGS`, separate them using commas (e.g. `team:backend,priority:high`).
+**Environment variable**: `DD_TAGS`<br/>
+**Default**: (none)<br/>
+**Example**: `team:backend`<br/>
+**Note**: Tags specified using `--tags` and via the `DD_TAGS` environment variable are merged. If the same key appears in both `--tags` and `DD_TAGS`, the value in the environment variable `DD_TAGS` takes precedence.
 
 `--max-concurrency`
-: The number of concurrent uploads to the API.<br>
+: The number of concurrent uploads to the API.<br/>
 **Default**: `20`
 
 `--dry-run`
-: Runs the command without the final upload step. All other checks are performed.<br>
+: Runs the command without the final upload step. All other checks are performed.<br/>
```

Comment:
@juan-fernandez does this actually do anything for junit upload? What does it print

Reply:
the files to be uploaded. It's something from sourcemaps. We can remove it if we don't think it adds value

Reply:
I have rephrased this a bit.